### PR TITLE
EXC-68: customize react scripts

### PR DESCRIPTION
**Pull request by josorioP44**

### CODE REVIEW

File: packages/react-scripts/config/webpack.config.js
Comment (adamnolte) [inline packages/react-scripts/config/webpack.config.js:211]: Can we use the name from the package.json here? Since this will be used in many different projects we don't want them all named voc.

File: packages/react-scripts/config/webpack.config.js
Comment (adamnolte) [inline packages/react-scripts/config/webpack.config.js:211]:
```suggestion
      filename: `${appPackageJson.name}.js`,
```

**Review by adamnolte — APPROVED** [review]

**Review by jeramiahcerda — COMMENTED** [review]
lgtm! Please get a review from Adam before merging.

**Review by adamnolte — APPROVED** [review]

**Pull request merged**
